features/tests/test_api.py

```diff
@@ -38,7 +39,7 @@ def api_client():
 
 
 @freeze_time("2019-12-16 12:00:01")
-def test_features_query(snapshot, api_client):
+def test_features_query_and_visibility(snapshot, api_client):
```

Comment:
I think the visibility could be tested separately, since it would make it easier to tell what might be broken if this test doesn't try to test two somewhat different things.

Reply:
tests split